mlir/lib/AsmParser/Parser.cpp

```diff
@@ -308,6 +309,51 @@ OptionalParseResult Parser::parseOptionalInteger(APInt &result) {
   return success();
 }
 
+namespace {
+bool isBinOrHexOrOctIndicator(char c) {
+  return (llvm::toLower(c) == 'x' || llvm::toLower(c) == 'b' ||
```

Comment:
Looking at the lexer, it seems like only hex is supported there. I think if you did a binary one, it would not be returned as a single integer.

Reply:
thanks!